src/Agent.Plugins/GitSourceProvider.cs

```diff
@@ -1315,6 +1305,32 @@ public async Task PostJobCleanupAsync(AgentTaskPluginExecutionContext executionC
             }
         }
 
+        public async void SetGitFeatureFlagsConfiguration(
```

Comment:
Can we re-use the same code from the `Agent.Worker/Build/GitSourceProvider.cs`?

Reply:
I would not reuse the method from Agent.Worker, since it is part of the Releases functionality. Also, paths are formed differently there.
Even if we think about reusing it, it would be better to avoid interaction with Git within Agent.Worker implementation and use methods from Agent.Plugin instead.
But I assume that this is a breaking change.